### DISCUSSION TITLE
Import relative modules in blocks

### DISCUSSION THREAD

**Discussion by michaeljcoyne** [area17/twill#1634] [General]
Is it possible to import relative modules in blocks

eg.
import { bus } from '../../../js/app'

My npm build script will compile but twill:dev throws error:

This relative module was not found:

Thanks

Michael

**Comment by haringsrob** [comment]
I have not worked with custom js in Twill yet. Maybe @pboivin has an answer?

**Comment by pboivin** [comment]
It should be possible to use relative imports. With the current custom component workflow, your files are copied to `vendor/area17/twill/frontend/js/components/customs/` as part of the build process. So, for a relative import from your `resources/js/app.js`, I think you would use:
```
import { bus } from '../../../../../../resources/js/app'
```
A bit unintuitive... This will surely be improved when Twill UI is refactored in the future.
